expression/builtin_string.go

```diff
@@ -679,9 +679,6 @@ func (b *builtinRepeatSig) evalString(row chunk.Row) (d string, isNull bool, err
 		return "", true, handleAllowedPacketOverflowed(b.ctx, "repeat", b.maxAllowedPacket)
 	}
 
-	if int64(byteLength) > int64(b.tp.Flen)/num {
```

Comment:
Could you explain a bit why this is removed?

Reply:
Query like this "select REPEAT(X'C385', 8777215)" will return null without message but mysql can return normally.